org.eclipse.xtext.ui.tests/src-longrunning/org/eclipse/xtext/ui/tests/editor/DirtyStateEditorSupportIntegrationTest.java

```diff
@@ -130,6 +130,11 @@ public void testUndoRedo() throws Exception {
 	 * @see https://github.com/eclipse-xtext/xtext/issues/2385
 	 */
 	@Test public void testModifyFileInExternEditor() throws Exception {
+		if (System.getProperty("os.name").startsWith("Mac")) {
```

Comment:
Maybe use an Assumption.assumeFalse instead?

Reply:
Just saw https://github.com/eclipse-xtext/xtext/pull/3283#pullrequestreview-2484804398

Reply:
But it will fail for all other cases or isn't it ? Did I get it wrong?

Reply:
Done..